ports/stm32/usbd_cdc_interface.c

```diff
@@ -157,6 +160,12 @@ int8_t usbd_cdc_control(usbd_cdc_state_t *cdc_in, uint8_t cmd, uint8_t *pbuf, ui
                 #endif
             } else {
                 cdc->connect_state = USBD_CDC_CONNECT_STATE_DISCONNECTED;
+                #if MICROPY_HW_USB_CDC_1200BPS_TOUCH
```

Comment:
did you want to enable this on portenta?

Reply:
It's already enabled.